docs/guides/tournaments/hosting.md

```diff
@@ -2,9 +2,11 @@
 title: Hosting a Tournament
 sidebar_position: 2
 description: Run your own Battlesnake Tournament for your team, classmates, or friends in just a few clicks. 
+displayed_sidebar: guides
```

Comment:
This page is linked to from TWO different sidebars now

This opts specifically into showing the Guides sidebar when on this page

This is mostly for Mobile where the sidebar nav only shows the current category by default